src/plugins/plot/stackedPlot/mixins/objectStyles-mixin.js

```diff
@@ -128,36 +128,26 @@ export default {
       }
     },
     updateStyle(styleObj) {
-      let elemToStyle = this.getStyleReceiver();
+      const elemToStyle = this.getStyleReceiver();
 
-      if (!styleObj || elemToStyle === undefined) {
+      if (!styleObj || !elemToStyle) {
         return;
       }
 
-      let keys = Object.keys(styleObj);
+      // handle visibility separately
```

Comment:
Why do we handle visibility outside of the `requestAnimationFrame` callback?

Reply:
This is a great question to which I don't have a good answer, I'll move it back in unless I remember why!

Reply:
so, the only benefit I could see is that visibility would be styled immediately, but I don't see that being a big enough benefit. It seems confusing to leave it out. I'll put it in the raf.